app/src/main/java/nerd/tuxmobil/fahrplan/congress/schedule/observables/ScrollToCurrentParameter.kt

```diff
@@ -0,0 +1,17 @@
+package nerd.tuxmobil.fahrplan.congress.schedule.observables
+
+import nerd.tuxmobil.fahrplan.congress.models.DateInfos
+import nerd.tuxmobil.fahrplan.congress.models.ScheduleData
+import nerd.tuxmobil.fahrplan.congress.schedule.FahrplanFragment
+import nerd.tuxmobil.fahrplan.congress.schedule.FahrplanViewModel
+
+/**
+ * Payload of the observable [scrollToCurrentParameter][FahrplanViewModel.scrollToCurrentParameter]
+ * property in the [FahrplanViewModel] which is observed by the [FahrplanFragment].
+ */
+data class ScrollToCurrentParameter(
```

Comment:
I would introduce new wording for the scroll action. I know it is kind of UI speak but I don't see using "scroll" **and** "focus" helps with the understanding. I did not append "session" because I believe it clear enough from the context where it is used.

Reply:
Fair enough regards the scrolling. In regards to name - I do not agree entirely - I had a bit of trouble to understand that right away, since I needed to figure out the context where it is used. A better name would have help, I think.

Reply:
The word "current" **describes both** - current time and current session. The app scroll to the vertical position based on the current time of the device. I named the class this way to have **consistent wording** together with the associated view model and fragment functions and property. Did you come up with a name which improves this?

Reply:
Don't get me wrong - it's not about `Current` it's about `...Current_Parameter`. When I read it the first time, it was super confusing, while I thought `Current_Parameter` is a thing, but what it actually means is `scrollToCurrent...` (aside what it's Object is). Also a reason, why I think an Interface would help, since you can arguably get rid of the suffix `Parameter`. 
In terms of the object - it's a bit hard to find the appropriate word, when session is not an option - and I bet, `meeting` or `gathering` are also not much helpful since they will just introduce another term (which in the end is just a synonym for session). `Appointment` would be a candidate, but still an extra term. So I am still thinking on this, but this as the first paragraph describes not my main issue here.

Reply:
:heavy_check_mark: I think appending "Session" here is fine. We should definitely not introduce a new synonym. I renamed the function `scrollToCurrentSession()`, property `scrollToCurrentSessionParameter` and class name `ScrollToCurrentSessionParameter` in cc9817e4710a50ab526c7f92f94ae7aa9d9c389e.

Reply:
Thanks much better!